### PR TITLE
Refactor Kubernetes docs to cover changes to our Helm Chart

### PR DESCRIPTION
**Pull request by cognifloyd**
- k8s_ha: Refactor intro paragraph and drop helm 2 reference
- k8s_ha: Refactor configurable values list
- k8s_ha: Reword a few sections for clarity.
- k8s_ha: Copy notes about deploying mongo/rabbitmq from chart readme
- k8s_ha: begin documenting st2packs vs volumes methods
- k8s_ha: reword description of using Shared Volumes
- k8s_ha: add more notes about Storage Volumes

### CODE REVIEW

File: docs/source/install/k8s_ha.rst [hunk 1]
```diff
@@ -52,16 +54,17 @@ or ``st2`` CLI client:
...
 
 Helm Values
 ___________
 Helm package ``stackstorm-ha`` comes with default settings (see `values.yaml <https://github.com/StackStorm/stackstorm-ha/blob/master/values.yaml>`_).
-Fine-tune them to achieve desired configuration for the StackStorm HA K8s cluster.
+Fine-tune them to achieve desired configuration for your StackStorm HA K8s cluster.
 
 .. note::
-    Keep custom values you want to override in a separated yaml file so they won't get lost.
+    Keep custom values you want to override in a separate yaml file so they won't get lost.
```
Comment (ericreeves) [inline docs/source/install/k8s_ha.rst:67]: This is rather pedantic, but typically YAML is capitalized.

File: docs/source/install/k8s_ha.rst [hunk 2]
```diff
@@ -312,7 +377,7 @@ Feedback Needed!
 ----------------
 As this deployment method new and beta is in progress, we ask you to try it and provide your feedback via
 bug reports, ideas, feature or pull requests in `StackStorm/stackstorm-ha <https://github.com/StackStorm/stackstorm-ha>`_,
-and ecourage discussions in `Slack <https://stackstorm.com/community-signup>`_ ``#docker`` channel or write us an email.
+and ecourage discussions in `Slack <https://stackstorm.com/community-signup>`_ ``#k8s`` channel or write us an email.
```
Comment (ericreeves) [inline docs/source/install/k8s_ha.rst:380]: Typo.  "encourage".

Comment (armab) [reply docs/source/install/k8s_ha.rst:380]: Also OK to drop "write us an email" part.

**Review by armab — APPROVED** [review]
LGTM

**Comment by cognifloyd** [comment]
v0.70.0 of the helm chart was just released: https://github.com/StackStorm/stackstorm-ha/releases/tag/v0.70.0
So, I'll merge this now too to document it :)

**Pull request merged**